metaflow/plugins/events_decorator.py

```diff
@@ -163,11 +205,23 @@ class TriggerOnFinishDecorator(FlowDecorator):
     @trigger_on_finish(flows=['my_project.branch.my_branch.FooFlow', 'BarFlow'])
     ```
 
+    You can also specify just the project or project branch (other values will be
+    inferred from the current project or project branch):
+    ```
+    @trigger_on_finish(flow={"name": "FooFlow", "project": "my_project", "project_branch": "branch"})
```

Comment:
any reason why this uses `project_branch` instead of `branch`? the CLI option is simply `--branch` so would it make sense to unify these?

Reply:
For posterity: the `--branch` and `project_branch` are actually slightly different thus the different name. If you do `--branch foo` for example, the `project_branch` should be either `prod.foo` or `test.foo`.